lib/tapioca/internal.rb

```diff
@@ -23,6 +23,7 @@
 require "thor"
 require "yaml"
 require "yard-sorbet"
+require "prism"
```

Comment:
I think it should be fine since we will always depend on `rbi` and in turn it will always depend on `prism`.

However, I would like to start a discussion about folding the `rbi` gem into Tapioca, since it's the only consumer of the gem. It would make maintenance a bit easier, fewer dependencies to manage, not need for RBI extensions in Tapioca and so on.